src/panels/config/automation/ha-automation-picker.ts

```diff
@@ -169,6 +171,11 @@ class HaAutomationPicker extends LitElement {
         )}
         hasFab
       >
+        <ha-icon-button
+          slot="toolbar-icon"
+          icon="hass:help-circle"
+          @click=${this._showHelp}
+        ></ha-icon-button>
```

Comment:
```suggestion
	    <mwc-icon-button @click=${this._showHelp}>
       	 	<ha-svg-button
        	  slot="toolbar-icon"
        	  .path=${mdiHelpCircle}
        	></ha-svg-button>
	    </mwc-icon-button>
```

This formatting is probably not perfect

Reply:
I pulled that code from both the scripts picker and the scenes picker. Are you saying both need to be updated?

Reply:
Yes. We recently moved to use this method with the new MDI/JS library

Reply:
I'm happy to change all of them, I just want to confirm that's what needs to be done

Reply:
Hmm. That change makes the icons disappear. I'm getting an error in the console 
`DevTools failed to load SourceMap: Could not parse content for http://10.1.1.4:8124/config/scene/mdc.circular-progress.min.css.map: Unexpected token < in JSON at position 0`

Reply:
That has nothing to do with that. Make sure to import the correct elements

Reply:
Sorry :/ I can't get the icon to show up again. I think I have all the imports it needs and I'm not seeing any errors. I'll try again tomorrow I suppose

Reply:
oh
I know what it is

Reply:
I fucked the comment up
its supposed to ha-svg-icon

Reply:
Ah yup, that was part of it. Thanks for catching that! The other thing appears to be `slot` needing to be in mwc-icon-button instead of inside ha-svg-icon